docs/index.md

```diff
@@ -0,0 +1,47 @@
+# Song Developer Documentation
+
+---
+
+**Navigation**
+- [Operation](./operation/operation.md)
+- [Contribution](./contribution/contribution.md) 
+
+---
+
+# Background
+
+Song is a metadata validation and tracking tool designed to streamline the management of genomics data across multiple cloud storage systems. With Song, users can create high-quality and reliable metadata repositories with minimal human intervention. As a metadata management system, Song does not handle file transfer and object storage. Song interacts with a required companion application, <a href="https://github.com/overture-stack/score" target="_blank" rel="noopener noreferrer">Score</a>, which manages file transfers and object storage.
```

Comment:
I think what you say here is correct but it does not clearly make the connection that Song is tracking the files you have uploaded in cloud storage system and managing the metadata associated with those files. Needs to be stated explicitly that this is about tracking files and managing their metadata.

Reply:
I think of Song as a File Catalog, like an inventory system that tracks what files we have and the meta data about them. Not sure if that verbiage is helpful.